packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@humeai/voice-react",
-  "version": "0.1.2-beta.1",
+  "version": "0.1.2-beta.2",
```

Comment:
reminder to bump the version for all 4 packages in `/packages` since we're keeping them in lock-step